### ISSUE TITLE
Inline feature

### ISSUE THREAD

**Issue by bostrot** [yukuku/telebot#54]
Is there any inline feature planned? 
I mean how can I answer to an InlineQuery with this python-based bot?
Would appreciate any help!

**Comment by suyashb95** [comment]
Yeah I'd like to use such a feature too. Right now I'm using [this](https://github.com/eternnoir/pyTelegramBotAPI) for inline queries.
